cairo/ethereum/cancun/bloom.cairo

```diff
@@ -11,7 +11,7 @@ from starkware.cairo.common.cairo_builtins import BitwiseBuiltin, KeccakBuiltin
 
 from ethereum_types.bytes import Bytes, BytesStruct, Bytes1DictAccess, Bytes32, TupleBytes32
 from ethereum_types.numeric import U256, Uint, U128
-from ethereum.utils.numeric import max, Uint_from_be_bytes, divmod, is_zero
+from ethereum.utils.numeric import max, Uint64_from_be_bytes, divmod, is_zero
```

Comment:
renaming this util as it tests number of bytes lower than 8 + takes be bytes as input